src/ore_algebra/analytic/deform.py

```diff
@@ -742,8 +742,7 @@ def xvor(self):
         # Additional edges between virtual vertices
         l = len(self.hull)
         edges.append((voff + l - 1, voff, -1)) # edge crossing the unbounded cut
-        for i in range(l - 1):
-            edges.append((voff + i, voff + (i + 1), -2-i))
+        edges.extend((voff + i, voff + (i + 1), -2-i) for i in range(l - 1))
```

Comment:
this one is intentional

Reply:
c'est à dire ? je suis pas sur de comprendre. Tu preferes la version en deux lignes ?